keygen/src/keygen.rs

```diff
@@ -131,6 +211,45 @@ fn grind_validator_starts_and_ends_with(v: String) -> Result<(), String> {
     Ok(())
 }
 
+fn acquire_language(matches: &ArgMatches<'_>) -> Language {
+    match matches.value_of("language").unwrap() {
```

Comment:
Since we have the ArgConstant above, might as well use it here
```suggestion
    match matches.value_of("LANGUAGE_ARG.name).unwrap() {
```

Reply:
Good points all.  I should have caught that.